examples/topic/topic.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: topic.kafka.crossplane.io/v1alpha1
 kind: Topic
 metadata:
-  name: sample-topic
+  name: topic2
 spec:
   forProvider:
     replicationFactor: 1
-    partitions: 3
+    partitions: 1
```

Comment:
It would be quite helpful to show how to make further configurations, maybe as [commented out](https://github.com/crossplane-contrib/provider-helm/blob/master/examples/sample/release.yaml#L17).

For example: 

```yaml
spec:
  forProvider:
    replicationFactor: 1
    partitions: 1
#    config:
#      compression.type: snappy
#      flush.ms: "1000"
```